utils/property-structs.stanza

```diff
@@ -47,29 +47,27 @@ public pcb-struct ocdb/utils/property-structs/PowerPin :
   
 ; Library property name: digital-input
 public pcb-struct ocdb/utils/property-structs/DigitalInput :
-  vil:Toleranced|RelativeVoltage
-  vih:Toleranced|RelativeVoltage
-  vdd-pin:JITXObject
-  gnd-pin:JITXObject
-  leakage-current:Double
+  vil:Toleranced|RelativeVoltage    ; Range of voltages that will read as logic low (V)
+  vih:Toleranced|RelativeVoltage    ; Range of voltages that will read as logic high (V)
+  vdd-pin:JITXObject                ; Pin that powers the high reference 
+  gnd-pin:JITXObject                ; Pin that powers the low reference
+  leakage-current:Double            ; Leakage current in Amps
```

Comment:
Sometimes leakage-current has a min and max like -1uA, +1uA so maybe a Toleranced() is better here?